src/base/rss/rss_autodownloadrule.cpp

```diff
@@ -393,6 +462,8 @@ AutoDownloadRule AutoDownloadRule::fromVariantHash(const QVariantHash &varHash)
     rule.setAddPaused(TriStateBool(varHash.value("add_paused").toInt() - 1));
     rule.setLastMatch(varHash.value("last_match").toDateTime());
     rule.setIgnoreDays(varHash.value("ignore_days").toInt());
+    rule.setUseSmartFilter(varHash.value("smart_filter", false).toBool());
```

Comment:
Right, but I had old settings that needed importing, and anyone who's been using this patch will also have settings that they might like importing (since remembering what's been downloaded is the point of this patch), it would be nice if this can be kept.